versions/1/elasticsearch/spec.yml

```diff
@@ -0,0 +1,28 @@
+spec:
+  additionalContents: false
+  contents:
+    - description: Folder containing ML model assets
+      type: folder
+      name: ml_model
+      required: false
+      contents:
+        - description: An ML model file
+          type: file
+          contentMediaType: "application/json"
+          pattern: '^{PACKAGE_NAME}-.+\.json$'
+    - description: Folder containing Elasticsearch Ingest Node pipeline definitions
+      type: folder
+      name: ingest_pipeline
+      additionalContents: false
+      contents:
+        - description: Supporting ingest pipeline definitions in YAML
```

Comment:
I guess you can also add test file for such ingest pipeline in `elasticsearch` directory.